onchain-non-merklized-issuer-base/v0/js/src/index.ts

```diff
@@ -0,0 +1,6 @@
+/* Autogenerated file. Do not edit manually. */
+/* tslint:disable */
+/* eslint-disable */
+export type { NonMerklizedIssuerBase } from './NonMerklizedIssuerBase';
+export * as factories from './factories';
```

Comment:
@Kolezhniuk @ilya-korotya do we need exports other than NonMerklizedIssuerBase?

Reply:
@vmidyllic if it's gonna be used - yes, it no - no. I guess it's more question to @ilya-korotya